### PR TITLE
Update rambler.ru

### PR DESCRIPTION
**Pull request by jfx2006**
Updated to new hostnames and changed password auth based on advertised capabilities.

SMTP:
220 mail.rambler.ru ESMTP ready
ehlo tgb.nt  
250-mail.rambler.ru
250-SIZE 104857600
250-ENHANCEDSTATUSCODES
250-8BITMIME
250 AUTH PLAIN LOGIN

POP:
+OK POP3 ready
CAPA
+OK Capability list follows
LAST
TOP
USER
PIPELINING
UIDL
IMPLEMENTATION nginx
SASL PLAIN LOGIN
.

IMAP:
* OK IMAP4 ready
1 CAPABILITY    
* CAPABILITY IMAP4rev1 UNSELECT LITERAL+ IDLE QUOTA AUTH=PLAIN
1 OK completed
DONE

### CODE REVIEW

File: ispdb/rambler.ru [hunk 1]
```diff
@@ -5,31 +5,31 @@
...
       <socketType>SSL</socketType>
       <username>%EMAILADDRESS%</username>
-      <authentication>password-encrypted</authentication>
+      <authentication>password-cleartext</authentication>
     </outgoingServer>
   </emailProvider>
-  <documentation url="http://help.rambler.ru/article.html?s=141&amp;id=41">
-    <descr lang="en">POP, Thunderbird, with screenshots</descr>
+  <documentation url="https://help.rambler.ru/mail/mail-pochtovye-klienty/1277/">
+    <descr lang="ru">How to set up Mozilla Thunderbird?</descr>
```
Comment (mkmelin) [inline ispdb/rambler.ru:30]: This text is in English, so it should be lang="en". Same below.

**Review by mkmelin — CHANGES_REQUESTED** [review]
Except for the lang setting, looks good!

File: ispdb/rambler.ru (hunk 1 above)
Comment (jfx2006) [reply ispdb/rambler.ru]: Fixed in follow-up commit.

**Comment by benbucksch** [comment]
> Except for the lang setting, looks good!

@mkmelin You are thinking of HTML, where the `lang=` attribute describes the language of the content of the element. This is XML and a more specific spec, and here, the `lang=` attribute describes the language of the linked document. So, the originally submitted patch was correct.

**Review by benbucksch — APPROVED** [review]

**Comment by benbucksch** [comment]
The lang attribute is the language of the web document, so "ru" is correct.

r=Ben as-is

Am 11. September 2022 06:03:06 MESZ schrieb Magnus Melin ***@***.***>:
***@***.*** requested changes on this pull request.
>
>Except for the lang setting, looks good!
>
>>      </outgoingServer>
>   </emailProvider>
>-  <documentation url="http://help.rambler.ru/article.html?s=141&amp;id=41">
>-    <descr lang="en">POP, Thunderbird, with screenshots</descr>
>+  <documentation url="https://help.rambler.ru/mail/mail-pochtovye-klienty/1277/">
>+    <descr lang="ru">How to set up Mozilla Thunderbird?</descr>
>
>This text is in English, so it should be lang="en". Same below.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/thundernest/autoconfig/pull/41#pullrequestreview-1103232450
>You are receiving this because you are subscribed to this thread.
>
>Message ID: ***@***.***>
-- 
Sent from my cell phone. Please excuse the brevity.

**Review by Sandrovvvs — COMMENTED** [review]
K

**Pull request merged**